kitchen.yml

```diff
@@ -19,33 +20,20 @@ verifier:
   - test/integration/default
 
 platforms:
-- name: mojave-chef17
-  driver:
-    box: microsoft/macos-mojave
-    box_version: 10.14.6
-  provisioner:
-    product_version: 17
-
-- name: catalina-chef17
-  driver:
-    box: microsoft/macos-catalina
-    box_version: 10.15.7
-  provisioner:
-    product_version: 17
-
-- name: big-sur-chef17
+- name: big-sur-x86
   driver:
     box: microsoft/macos-big-sur
-    box_version: 11.6.5
-  provisioner:
-    product_version: 17
+    box_version: 11.7.8
 
-- name: monterey-chef17
+- name: monterey-x86
   driver:
     box: microsoft/macos-monterey
-    box_version: 12.6.3
-  provisioner:
-    product_version: 17
+    box_version: 12.6.7
+
+- name: ventura-arm
```

Comment:
Do we want to test ventura-x86 as well?

Reply:
Still working on getting those boxes working on our Ventura hosts before we open those floodgates, but yea we can add that suite and leave it off of the pipelines.yml